### PR TITLE
Add title and buttons to admin and mod log

### PR DESCRIPTION
**Pull request by albertlast**
Base is for me the error log:
grafik

Atm the mod log look like this:
grafik
we notice missing title and missing button on right upper side
with the pr it's look like this:
grafik

### CODE REVIEW

**Comment by StealthWombat** [comment]
Non-critical, but a good idea.

**Comment by dragomano** [comment]
It should to add `loadLanguage('Admin+Modlog');` into Line 46 to prevent possible errors.

**Comment by albertlast** [comment]
under which circumstance the line is needed?

**Comment by dragomano** [comment]
1. Go to Moderate -> Moderation log
2. Then we see a new undefined error in the Error log:
`Undefined index: moderation_log`
3. Replace `loadLanguage('Modlog');` with `loadLanguage('Admin+Modlog');` and we do not see this error anymore.

**Comment by MissAllSunday** [comment]
@albertlast Can you add the change @dragomano suggested?

File: Sources/Modlog.php [hunk 1]
```diff
@@ -163,6 +163,7 @@ function ViewModlog()
 	// This is all the information required for a watched user listing.
 	$listOptions = array(
 		'id' => 'moderation_log_list',
+		'title' => $context['log_type'] === 3 ? $txt['admin_log'] : $txt['moderation_log'],
```
Comment (MissAllSunday) [inline Sources/Modlog.php]: Since we historically haven't perform strict checks and due to the volatile nature of global vars, please use == insead of ===

**Review by MissAllSunday — CHANGES_REQUESTED** [review]

**Review by MissAllSunday — APPROVED** [review]

**Comment by MissAllSunday** [comment]
Thanks!

**Pull request merged**